<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>930</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>IBM Plex Sans KR</family>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>880</width>
      <height>830</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_right">
     <attribute name="title">
      <string>Right Approach</string>
     </attribute>
     <widget class="QRadioButton" name="male_right">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="female_right">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
     <widget class="QLabel" name="input_map_right">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>450</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="picc_app.qrc">:/images/graphic/Rt_none.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>351</width>
        <height>186</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="hvd_right_label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Humero-Vertebral Distance</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="rhd_right_label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>2nd Ribs Horizontal Distance</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="hvd_right">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="inputMask">
          <string>##.###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="vbu_right_label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Two Vertebral Body Units</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="tcd_right_label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Thoraco-Carinal Distance</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="tcd_right">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="inputMask">
          <string>##.###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="vbu_right">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="inputMask">
          <string>##.###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="rhd_right">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="inputMask">
          <string>##.###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ccl_right_label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Contralateral Clavicle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ccl_right">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="inputMask">
          <string>##.###</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="clear_right">
         <property name="font">
          <font>
           <family>Arial</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Variables on AP-CXR</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>The Result</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>360</y>
        <width>660</width>
        <height>420</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="picc_app.qrc">:/images/graphic/Rt_Result.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>181</width>
        <height>140</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="result_right_plain">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Plain</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="result_right_cook">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="result_right_genoss">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="result_right_navilyst">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Navilyst</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>GENOSS</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Cook</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>BD</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="result_right_bd">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="cp_right">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>430</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>##.#</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>IBM Plex Sans KR</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>410</y>
        <width>181</width>
        <height>110</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Plain</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="ecl_right_plain">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_24">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Navilyst</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_25">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Cook</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>BD</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>GENOSS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="ecl_right_bd">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="ecl_right_cook">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="ecl_right_navilyst">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="ecl_right_genoss">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a name=&quot;docs-internal-guid-ad46f569-7fff-b819-d061-3cf501e4ccba&quot;/&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:400; color:#000000; background-color:transparent;&quot;&gt;i&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:400; color:#000000; background-color:transparent;&quot;&gt;f the procedure is performed in such a way that the catheter hub touches the puncture site so that there is no catheter exposed outside the body,&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:400; color:#ff0000; background-color:transparent;&quot;&gt; the distance from the hub to the 0 point&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:400; color:#000000; background-color:transparent;&quot;&gt; must be additionally subtracted and applied as eCL. These values are all different depending on the manufacturer.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>CP+eCL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_left">
     <attribute name="title">
      <string>Left Approach</string>
     </attribute>
     <widget class="QLabel" name="input_map_right_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>450</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="picc_app.qrc">:/images/graphic/Lt_none.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>850</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="picc_app.qrc">:/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>male_right</tabstop>
  <tabstop>female_right</tabstop>
  <tabstop>clear_right</tabstop>
  <tabstop>ccl_right</tabstop>
  <tabstop>rhd_right</tabstop>
  <tabstop>hvd_right</tabstop>
  <tabstop>tcd_right</tabstop>
  <tabstop>vbu_right</tabstop>
  <tabstop>cp_right</tabstop>
 </tabstops>
 <resources>
  <include location="picc_app.qrc"/>
 </resources>
 <connections/>
</ui>
